.github/workflows/triage-issue.yml

```diff
@@ -0,0 +1,19 @@
+name: Triage Issue
+on:
+  issues:
+    types:
+      - reopened
+      - opened
+jobs:
+  label_issues:
+    runs-on: windows-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Label issues
+        uses: andymckay/labeler@1.0.3
+        with:
+          add-labels: "NEEDS TRIAGE"
```

Comment:
Change the value of the `add-labels` key to the suggestion below.

```suggestion
          add-labels: "⚕️NEEDS TRIAGE"
```

---

Just a quick tip.  To be able to use labels with emoji's, you have to include the emoji as part of the label name.  If you do not, then the first time that this workflow runs, it will detect that the label `NEEDS TRIAGE` does not exist and it will create one.  This would result in 2 labels existing which would be `NEEDS TRIAGE` AND `⚕️NEEDS TRIAGE`

Hopes this makes sense!!  Let me know if you need assistance. 

![image](https://user-images.githubusercontent.com/85414302/178301406-b5dfbeea-b26d-40f3-97b4-41fd8eec1a84.png)